requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/VIP-LES/EosLib@main#egg=EosLib  # TODO: change to commit hash or tag
+git+https://github.com/VIP-LES/EosLib@v0.3.0#egg=EosLib
```

Comment:
latest version is 0.4.1 -- you'll have to force update using the command on the EosPayload readme